packages/chain-adapters/src/evm/bnbsmartchain/BscChainAdapter.ts

```diff
@@ -56,39 +57,57 @@ export class ChainAdapter extends EvmBaseAdapter<KnownChainIds.BnbSmartChainMain
     return this.assetId
   }
 
+  async getGasFeeData(): Promise<GasFeeDataEstimate & { baseFeePerGas?: string }> {
```

Comment:
expose `baseFeePerGas` to accurately calculate the `maxPriorityFeePerGas` min fee diff.